### PR TITLE
text.trimmed implemented on eo

### PR DESCRIPTION
**Pull request by EugeneDar**
https://github.com/objectionary/eo-strings/issues/97

### CODE REVIEW

**Comment by EugeneDar** [comment]
In this implementation `text.trimmed` I implemented only `left-trimmed` and created puzzle finished it.

**Comment by EugeneDar** [comment]
@Graur @levBagryansky please check.

File: src/main/eo/org/eolang/txt/text.eo [hunk 1]
```diff
@@ -38,7 +38,47 @@
...
+          as-bytes.
+            s
+      *
+      [a i x]
+        up-cased.ascii > code!
+          x
+        if. > @
+          eq.
+            code
+            32
```
Comment (levBagryansky) [inline src/main/eo/org/eolang/txt/text.eo:57]: Does it mean 
`up-cased.ascii " "`
?

**Review by levBagryansky — CHANGES_REQUESTED** [review]
Nice!
Maybe its better to move `up-cased.ascii` into separate attribute?

File: src/main/eo/org/eolang/txt/text.eo (hunk 1 above)
Comment (EugeneDar) [reply src/main/eo/org/eolang/txt/text.eo:57]: @levBagryansky I'll fix it

**Comment by EugeneDar** [comment]
@levBagryansky please check again

**Review by levBagryansky — APPROVED** [review]
I like it

**Comment by EugeneDar** [comment]
@yegor256 please have a look

**Comment by yegor256** [comment]
@rultor merge

**Comment by EugeneDar** [comment]
@yegor256 please merge it again

**Comment by EugeneDar** [comment]
@yegor256 please, merge it again

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**